html/index.html

```diff
@@ -78,59 +82,70 @@ <h2>Software developer</h2>
       <div class="works_head">
         <h2 class="work_title">MY RECENT WORK</h2>
         <!--divider-->
-        <img alt="divider vector" src="../img/Vector.svg" class="divider_c">
+        <div class="divider_c"></div>
       </div>
+
       <div class="works_container">
```

Comment:
[OPTIONAL]
- You implemented the desktop version in a good way. Let's make it better by reducing the margin between sections.
What do you think? I will also  recommend reducing the margin between the projects.
<img width="701" alt="image" src="https://user-images.githubusercontent.com/77190203/150660074-ffe02f68-729c-4510-9231-f2d29f9a3c63.png">